prisondersdilemma.py

```diff
@@ -0,0 +1,202 @@
+image_02 = ImageStim(win, image="im_02.jpg") 
+image_03 = ImageStim(win, image="im_03.jpg") 
+image_04 = ImageStim(win, image="im_04.jpg") 
+image_05 = ImageStim(win, image="im_05.jpg") 
+image_06 = ImageStim(win, image="im_06.jpg")
+image_07 = ImageStim(win, image="im_07.jpg")
+image_08 = ImageStim(win, image="im_08.jpg")
+image_09 = ImageStim(win, image="im_09.jpg")
+image_10 = ImageStim(win, image="im_10.jpg")
+
+
+opponent = Imagetim () #for later in trial loop maybe setIm (if that exists) for randomly assigning opponent based on condition
+
+# txt within game
+connecttxt = TextStim ()
+cooptxt = TextStim ()
+deftxt = TextStim ()
+
+#log file
+log = open ('logfileprisonersdilemma.txt', 'w')
+log.write(' .... ') #insert headers of what should be logged
+
+#response keys
+resp = getKeys(keyList = ('q', 'space', 'left', 'right'))
+
+strategy = [1,2,3,4,5]  #random assignment of opponents strategy
+shuffle(strategy)
+
+randostrategy = [1,2] #for strategy 3
+
+
+condlist = 
+n_trials =  #need to set that
+
+
+#starting the game
+background_image.draw()
+win.flip()
+
+
+#trial loop
+for i in range(n_trials):
+	
+	#strategies
+
+	#opponent always cooperates
+	if strategy == 1:
+		opponent.draw()
+		cooptxt.draw()
+		win.flip()
+		wait(4) #seconds
+		connecttxt.draw
+		#sentence draw 
+		win.flip()
+	
+	#opponent always defects
+	elif strategy == 2:
+		opponent.draw()
+		deftxt.draw()
+		win.flip()
+		wait(4) #seconds
+		connecttxt.draw
+		#sentence draw 
+		win.flip()
+	
+	#random coop or defect
+	elif strategy == 3:
+		shuffle(randostrategy)
+		if randostrategy == 1:
+			opponent.draw()
+			cooptxt.draw()
+			win.flip() 
+			wait(4)#seconds
+			#sentence draw 
+			win.flip()
+		elif: 
+			opponent.draw()
+			deftxt.draw()
+			win.flip()
+			wait(4) #seconds
+			connecttxt.draw
+			#sentence draw 
+			win.flip()
+	
+	elif strategy == 4:
+		if condlist[i] == 1:
+			opponent.draw()
+			cooptxt.draw()
+			win.flip() 
+			wait(4)#seconds
+			connecttxt.draw
+			#sentence draw 
+			win.flip()
+		elif condlist[i] > 1:
+			if resp == 'right':
+				opponent.draw()
+				cooptxt.draw()
+				win.flip() 
+				wait(4)#seconds
+				connecttxt.draw
+				#sentence draw 
+				win.flip()
+			elif resp == 'left':
+				opponent.draw()
+				deftxt.draw()
+				win.flip() 
+				wait(4)#seconds
+				connecttxt.draw
+				#sentence draw 
+				win.flip()
+	elif strategy == 5:
+		if condlist[i] == 1:
+			opponent.draw()
+			deftxt.draw()
+			win.flip() 
+			wait(4)#seconds
+			connecttxt.draw
+			#sentence draw 
+			win.flip()
+		elif condlist[i] > 1:
+			if resp == 'right':
+				opponent.draw()
+				cooptxt.draw()
+				win.flip() 
+				wait(4)#seconds
+				connecttxt.draw
+				#sentence draw 
+				win.flip()
+			elif resp == 'left':
+				opponent.draw()
+				deftxt.draw()
+				win.flip() 
+				wait(4)#seconds
+				connecttxt.draw
+				#sentence draw 
+				win.flip()
+				
+		
+			
+
+
+
+
+
+
+#exit: 
+if resp == ['q']:
+	break 
```

Comment:
I know that you're not finished yet but perhaps just implement a short message for the user to understand the outcome and that displays long they played for instance :-)